assets/cubyz/biomes/ocean/shallows.zig.zon

```diff
@@ -1,53 +1,49 @@
 .{
 	.properties = .{
-		.cold,
 		.ocean,
+		.wet,
 	},
-	.type = .arctic_ocean,
-	.minHeight = -100,
-	.maxHeight = -22,
-	.radius = 500,
-	
-	.roughness = 20,
-	.hills = 10,
-
+	.radius = 128,
+	.minHeight = -64,
+	.maxHeight = -32,
+	.chance = 10,
+	.roughness = 15,
+	.hills = 15,
 	.music = "cubyz:under_the_water_sky",
-
 	.ground_structure = .{
-		"1 to 2 cubyz:cobblestone",
+		"0 to 1 cubyz:gravel",
+		"0 to 2 cubyz:cobblestone",
 	},
 	.structures = .{
 		.{
 			.id = "cubyz:ground_patch",
 			.block = "cubyz:mossy_cobblestone",
-			.chance = 0.064,
-			.width = 6,
-			.variation = 2,
-			.depth = 2,
-			.smoothness = 0.2,
-		},
-		.{
-			.id = "cubyz:ground_patch",
-			.block = "cubyz:gravel",
-			.chance = 0.032,
-			.width = 6,
-			.variation = 2,
+			.chance = 0.05,
+			.width = 12,
+			.variation = 4,
 			.depth = 2,
 			.smoothness = 0.2,
 		},
 	},
 	.transitionBiomes = .{
 		.{
-			.id = "cubyz:beach",
+			.id = "cubyz:ocean/cliffs",
+			.chance = 0.25,
+			.width = 2,
+			.properties = .{.antiMountain, .ocean},
+		},
+		.{
+			.id = "cubyz:ocean/beach",
 			.chance = 1,
 			.width = 1,
 			.properties = .{.land, .inland},
 		},
 		.{
-			.id = "cubyz:ocean_shelf",
+			.id = "cubyz:ocean/shelf",
 			.chance = 1,
+			.keepOriginalTerrain = 0.1,
```

Comment:
I think this might break the transition in some cases. But you should test it out if that really is the cause.